public/locales/es/gamepage.json

```diff
@@ -231,7 +231,8 @@
         "store": "Página de la tienda",
         "verify": "Verificar y reparar",
         "extraInfo": "Información extra",
-        "logs": "Registros"
+        "logs": "Registros",
+        "manageCategories": "Administrar Categorías"
```

Comment:
we don't add translations for languages other than english here in the code, the spanish translations should be later added to weblate once this PR is merge

this file can be overrode by weblate if we only have change here